### DISCUSSION TITLE
Name of repositories

### DISCUSSION THREAD

**Discussion by LeszekWrobel** [community/community#21860] [New to GitHub]
<p>Hi</p>
<p>How can I change the name of new repository ?</p>
<p>How can I deleted it ?</p>
<p>Leszek</p>

**Comment by tlystad24** [comment, marked as answer]
<p>Hello Leszek!</p>
<p> To rename a repository:</p>
<ol>
<li>Go to your repository</li>
<li>Navigate to the settings icon.</li>
<li>Edit the “Repository name” field on the top of the settings page and press “rename”.</li>
</ol>
<p>To delete a repository:</p>
<ol>
<li>
<p>Go to your repository</p>
</li>
<li>
<p>Navigate to settings</p>
</li>
<li>
<p>On the bottom of the settings page, in the “danger zone” press “Delete this repository”. You have to type the name of the repository to verify that you want to delete it.</p>
</li>
</ol>
<p>You can visit <a href="https://help.github.com/" rel="nofollow noopener">https://help.github.com/</a> for more information.</p>

**Comment by LeszekWrobel** [comment]
<p>Hi</p>
<p>Maybe it is stupid qestions for the  many GitHab users,  but you relly helped me.</p>
<p>Thank you.</p>
